README.md

```diff
@@ -1040,7 +1040,7 @@ constructor(options: KoncordeOptions = null)
 
 | Name | Type | Default |Description                      |
 |------|------|---------|---------------------------------|
-|`maxMinTerms`| `Number` | `256` | The maximum number of conditions a filter can hold after being canonicalized in its [CDNF](https://en.wikipedia.org/wiki/Canonical_normal_form) form. It is advised to test performances and memory consumption impacts before increasing this value. If set to 0, no limit is applied. |
+|`maxConditions`| `Number` | `50` | The maximum number of conditions a filter can hold. It is advised to test performances and memory consumption impacts before increasing this value. If set to 0, no limit is applied. |
```

Comment:
You should add a note saying that the limit is applied after normalization

Reply:
The wiki mentions that. As discussed a few minutes ago, I'll submit another PR removing most of the README's content and redirect to the wiki.